src/js/Controllers/UIController.js

```diff
@@ -242,6 +298,11 @@ class UIController {
         }
         this.onSystemContext("MAIN", context)
     }
+    onSubtleAlertTimeout(msgID, appID) {
+        delete this.timers[msgID];
+        store.dispatch(closeAlert(msgID, appID));
+        this.listener.send(RpcFactory.SubtleAlertErrorResponse(msgID, 10, 'subtle alert timed out'));
```

Comment:
Please use a constant or add a note stating that result code 10 refers to TIMED_OUT

Reply:
Marking as resolved for this PR. Please refer to https://github.com/smartdevicelink/generic_hmi/pull/287#discussion_r469189664